apps/account-worker/src/transaction_publisher/publisher.service.ts

```diff
@@ -196,6 +204,28 @@ export class TransactionPublisherService extends BaseConsumer implements OnAppli
     }
   }
 
+  async processProxyTxn(
+    ext: SubmittableExtrinsic<'promise', ISubmittableResult>,
+    accountId: string,
+    signature: HexString,
+  ): Promise<[SubmittableExtrinsic<'promise'>, HexString]> {
+    try {
+      const prefixedSignature: SignerResult = { id: 1, signature };
```

Comment:
is this the same as `SpRuntimeMultiSignature` ?

If that is the case then one other way we created these were like the following example

```rust
const proof = { Sr25519: u8aToHex(delegatorKeys.sign(u8aWrapBytes(payload.toU8a()))) };
```